terraform/main.tf

```diff
@@ -0,0 +1,92 @@
+provider "google" {
+    version = "3.5.0"
+
+    credentials = file("application_default_credentials.json")
+
+    project = "virustotal-step-2020"
+    region = "europe-west1"
+    zone = "europe-west1-c"
+}
+
+resource "google_compute_network" "vpc_network" {
+  name = "terraform-network"
+}
+
+
+resource "google_compute_instance" "vm_instance_1" {
+  name         = "terraform-instance-1"
```

Comment:
terraform-instance-1 maybe is a little bit obfuscated. I would use a more descriptive name